dts/bindings/base/base.yaml

```diff
@@ -19,6 +19,18 @@ properties:
     required: true
     description: compatible strings
 
+  hardware-id:
```

Comment:
How do you use it?

Reply:
Is this change related to your other PR?

Reply:
This is required for ACPI since acpi devices will be enumerated based on it.